core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/accountlist/AccountListViewModel.kt

```diff
@@ -91,7 +93,11 @@ class AccountListViewModel(
     private val dontSetCurrentSession: Boolean = savedStateHandle[ARG_DONT_SET_CURRENT_SESSION]
         ?.toBoolean() ?: false
 
+    private val shareAppUseCase: ShareAppUseCase? by instanceOrNull()
+    val sendAppOptionVisible = shareAppUseCase != null
```

Comment:
This is **not** following MVVM pattern. 

MVVM works as per coding style / docs: the UiState object has properties that are then observed by the view. shareAppOptionVisible should be a property of the AccountListUiState - set by the viewmodel - like all other properties.